### PR TITLE
provider/openstack Convert FixedIPS from struct to map for ResourceData

### PR DESCRIPTION
**Pull request by Sheile**
IMO, ResourceData can't handle list of native structure.

### CODE REVIEW

File: builtin/providers/openstack/resource_openstack_networking_port_v2.go [hunk 1]
```diff
@@ -90,6 +91,7 @@ func resourceNetworkingPortV2() *schema.Resource {
 						"ip_address": &schema.Schema{
 							Type:     schema.TypeString,
 							Required: true,
```
Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_networking_port_v2.go]: This needs changed to Optional: true

**Comment by jtopjian** [comment]
Looks like the one item needs fixed. That should clear up the CI tests.

**Comment by Sheile** [comment]
Sorry about that.
I rebased this branch from master which was merged with #4850.

I'll combine these pull requests next time.

Thanks!

**Comment by jtopjian** [comment]
No problem at all about the two PRs!

This looks good - thanks!

**Comment by jtopjian** [comment]
(Side note: CHANGELOG will not be updated for this PR since it closely ties into #4850)

**Pull request merged**